src/psyclone/core/access_info.py

```diff
@@ -250,20 +260,20 @@ def all_accesses(self):
         return self._accesses
 
     def add_access_with_location(self, access_type, location, node,
-                                 component_indices=None):
+                                 component_indices):
```

Comment:
Yes ;) I thought about that. I decided to remove this, because this simple class is only used internally (by `VariablesAccessesInfo`). When we automatically create component_indices in `VariablesAccessInfos`, we have access to the signature, i.e. we can make sure to create a consistent `ComponentIndices` object. In this class however, we don't have access to the signature, and therefore we can't guarantee that we will create a consistent set (e.g.: 'field%global%startx' would create `[[]]``, not `[[][][]]`.
Since the class will not be created by an application (I can't see a good reason, an application will always use `VariablesAccessInfos` as far as I can see), I disabled the default to guarantee we always have consistent info - and accepted the fact that I needed to be more verbose in the tests.

Reply:
OK, that's good enough for me :-)